SlackBotPrototype/API.Tests/EmbeddedResourceTestBase.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace API.Tests
+{
+	public class EmbeddedResourceTestBase

```

Comment:
since this doesn't use any instance variables or configure any other aspect of the class, should we look at just having it be either a mixin or a utility function? what are the downsides or upsides of using a class to do this?